src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-otel.mdx

```diff
@@ -118,6 +114,21 @@ To stop monitoring a Kubernetes cluster with OpenTelemetry, run this command:
     helm uninstall nr-k8s-otel-collector -n newrelic
 ```
 
+
+## Reduce data ingest [#reduce-data-ingest]
+
+The `LowDataMode` option is enabled by default to ingest only the metrics required by our Kubernetes UIs.
+
+If you need to cut down even more on data ingestion, you can just increase the scrape interval in the [`nr-k8s-otel-collector` chart values](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector#values) for each deployed component.
```

Comment:
"If you need to cut down even more on data ingestion,  increase the scrape interval in the..."

I prefer to be more direct so I would have dropped extra words like "You can just"

Reply:
Nice catch! I'll make that change.